{"name": "Получить заказ по его номеру. Попытка получить заказ по номеру без номера возвращает корректное сообщение об ошибке", "status": "passed", "description": "тест ручки \"Получить заказ по его номеру\"", "start": 1724853583877, "stop": 1724853583917, "uuid": "703e336f-6907-4bd4-9826-4cb5751dedad", "historyId": "653e320c616dff1b6bbfa17bf9606113", "testCaseId": "653e320c616dff1b6bbfa17bf9606113", "fullName": "tests.TestGetOrderByNumber#test_get_order_without_order_number_returns_error", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestGetOrderByNumber"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "31284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}